cmake/modules/FindSDL2.cmake

```diff
@@ -75,6 +75,7 @@ FIND_PATH(SDL2_INCLUDE_DIR SDL.h
         $ENV{SDL2DIR}
         PATH_SUFFIXES include/SDL2 include
         PATHS
+		C:/Programming/Libraries/SDL2-2.0.10/SDL2/include
```

Comment:
These are a bit hacky but seemed to be the only way to get my MSVC to find SDL2.